backend/main/src/Group/Domain/Entities/Group.ts

```diff
@@ -155,13 +156,26 @@ export class Group extends AggregateRoot<string, IGroupProps> {
  * This class is the group's ID.
  */
 export class GroupId extends Identifier<string> {
-	static create(id?: string): Result<GroupId, GroupIdDomainError> {
-		if (id) {
-			return Ok(new GroupId(id));
+	/**
+	 * the group ID is UUID.
+	 * @param id
+	 */
+	static create(id: string): Result<GroupId, GroupIdDomainError> {
+		if (!uuidValidate(id)) {
```

Comment:
Would you mind if I ask you to merge the develop branch and use the following function? Its function also checks if the argument is valid UUID V4.

https://github.com/genesis-tech-tribe/nishiki-backend/blob/develop/backend/main/src/Shared/Utils/Validator.ts

Reply:
@hitohata 
I see! I'll do that before merging